CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Corso-generated .meta files and permissions no longer appear in the backup details.
+- repo.BackupDetails from the SDK now filter out meta and dirmeta files, and empty folders, by standard.
```

Comment:
Do we want to leave "OneDrive files ending in `.meta` or `.dirmeta` are omitted from details and restores." in the known issues section?

Reply:
Yes, both conditions are true.  We are filtering out our meta files.  And also, we might filter out meta files that we did not create, unexpectedly.